Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -607,6 +650,23 @@ bool InspectorOverlay::removeGridOverlayForNode(Node& node)
     });
 }
 
+void InspectorOverlay::showSpecialGridOverlayForNode(Node& node, const InspectorOverlay::Grid::Config& gridOverlayConfig)
```

Comment:
Same question here about naming using `special` instead of something more descriptive?